roles/ovirt-repositories/README.md

```diff
@@ -0,0 +1,65 @@
+
+oVirt Repositories
+==================
+
+The `ovirt-repositories` role is used to set the repositories required for oVirt engine installation.
+By default it copies content of /etc/yum.repos.d/ to /tmp/repo-backup-{{timestamp}}.
+Currently it works only when you provide the release RPM.
+
+
+Requirements
+------------
+
+ * oVirt Python SDK version 4
+ * Ansible version 2.4
+
+Role Variables
+--------------
+
+| Name                  | Default value         |  Description                              |
+|-----------------------|-----------------------|-------------------------------------------|
+| ovirt_rpm_repo        | UNDEF                 | URL of RPM package with repository files. |
+| use_subscription_manager| False | If true it will use repos from subscription manager. |
+| rh_username | UNDEF | Username to use for subscription manager. |
+| rh_password | UNDEF | Password to use for subscription manager. |
+| pool_ids | UNDEF | List of pools ids to subscribe to. |
+
+
+Dependencies
+------------
+
+No.
+
+Example Playbook
+----------------
+
+```yaml
+---
+- name:  install RPM with repository files
+  hosts: localhost
+
+  vars:
+    ovirt_rpm_repo: http://resources.ovirt.org/pub/yum-repo/ovirt-master-release.rpm
+
+  roles:
+    - role: ovirt-repositories
+
+- name:  install RPM with repository files
+  hosts: localhost
+
+  vars:
+    use_subscription_manager: True
+    rh_username: myuser
+    rh_password: mypassword
```

Comment:
please no plain text password, let's follow best practise using vault, so people don't copy paste this